scout/server/blueprints/variants/controllers.py

```diff
@@ -1024,21 +1100,17 @@ def generate(header, lines):
     )
 
 
-def download_variants(store, case_obj, variant_objs):
+def download_variants(
+    store: MongoAdapter, case_obj: dict, variant_objs: CursorType, category: Optional[str] = None
+) -> Response:
     """Download filtered variants for a case to an excel file
 
-    Args:
-        store(adapter.MongoAdapter)
-        case_obj(dict)
-        variant_objs(PyMongo cursor)
-
     Returns:
         an HTTP response containing a csv file
     """
-    document_header = variants_export_header(case_obj)
-    export_lines = []
+    document_header = variants_export_header(case_obj, category)
     export_lines = variant_export_lines(
-        store, case_obj, variant_objs.limit(EXPORTED_VARIANTS_LIMIT)
+        store, case_obj, variant_objs.limit(EXPORTED_VARIANTS_LIMIT), category
```

Comment:
Yes, category might really be the key to simplify the export. But I would keep that code outside this PR